compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -1448,6 +1450,55 @@ fn expr_as_constructor(
     option(return_type, option_value)
 }
 
+// fn as_for(self) -> Option<(Quoted, Expr, Expr)>
+fn expr_as_for(
```

Comment:
We should probably start changing names to be returned as `CtString`s but we can do this in another PR.